### DISCUSSION TITLE
How access ~scope from another Brython script ?

### DISCUSSION THREAD

**Discussion by BobSquarePants** [brython-dev/brython#2335] [Q&A] [answered]
Hi,

let say in a html page I've got the following

```html
<script type="text/python" src="/A.Brython"></script>
<script type="text/python" src="/B.Brython" id="ScriptB"></script>
```

in **B.Brython**
```python
def fnB():
    print('Hi from B')
```

is it possible from **A.Brython** to reach Functions. variables etc.. from **B.Brython** ?
https://brython.info/static_doc/3.12/en/jsobjects.html

I've tried:

in **B**

```python
window.fnB = fnB
```
it work but seem not the best way..

I've tried from **A**

```python
__BRYTHON__.getPythonModule('ScriptB').fnB()
```
but this give me an `Uncaught __dir__: ` error..

Any ideas ? thanks.

**Comment by denis-migdal** [comment, marked as answer]
> is it possible from **A.Brython** to reach Functions. variables etc.. from **B.Brython** ? https://brython.info/static_doc/3.12/en/jsobjects.html

Did you try something like :
```python
from ScriptB import fnB
```

**Reply by BobSquarePants** [reply]
Thanks @denis-migdal it work pretty good :)
